txs/payload/identify_tx.go

```diff
@@ -0,0 +1,40 @@
+package payload
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/burrow/crypto"
+	"github.com/hyperledger/burrow/execution/registry"
+)
+
+func NewIdentifyTx(address crypto.Address, node *registry.RegisteredNode) *IdentifyTx {
+	return &IdentifyTx{
+		Input: &TxInput{
+			Address: address,
+		},
+		Node: node,
+	}
+}
+
+func (tx *IdentifyTx) WithSig(bs []byte) *IdentifyTx {
```

Comment:
I don't necessarily object to `Sig` as an abbreviation a priori, but everywhere else we use `Signature` and we also have `Signatory` so can we be consistent here and not abbreviate?